.circleci/config.yml

```diff
@@ -61,6 +61,35 @@ commands:
             chmod +x grabpl
             mv grabpl /tmp
 
+  halt-if-not-matching-files:
+    description: "Requires files of last commit to match provided glob pattern. If no matches are found, the step will halt and be marked as successful."
```

Comment:
<3

Perhaps we can check all commits since the branch was created?

`git log --oneline master...myBranch``

Reply:
The current approach should already check all commits in branch compared to origin/master `git log --oneline --pretty="format:" --name-only origin/master..`

Reply:
But the description says only last commit, so the description wrong? `Requires files of last commit to match provided glob pattern`